### PR TITLE
8512 - added new svgs for hero section; added text and buttons and st…

### PR DESCRIPTION
**Pull request by brianpetway**
…yles for the hero section

**High level description:**

Updating the hero section of the homepage.

**Technical details:**

I added the new svgs, text, buttons, and styles for the hero section. Mocks and Zeplin are linked in the ticket and below.

**JIRA Ticket:**
[DEV-8512](https://federal-spending-transparency.atlassian.net/browse/DEV-8512)

**Mockup:**
[Homepage MVP Desktop (invisionapp.com)](https://bahdigital.invisionapp.com/share/M8IAIDK7W4C#/screens?browse)

https://app.zeplin.io/project/61f8076f4ef77f11fcc37f55/dashboard?sid=621fa8fb092ccca26152dae8

The following are ALL required for the PR to be merged:

Author:
- [ x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [ x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [ x] Verified mobile/tablet/desktop/monitor responsiveness
- [ x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

File: src/js/components/homepageUpdate/hero/Hero.jsx [hunk 1]
```diff
@@ -4,7 +4,71 @@
...
+                            <div className="hero__pause-button-container">
+                                <p>pause button placeholder</p>
+                            </div>
+                        </div>
+                    </div>
+                    <div className="hero__right-image-wrapper">
+                        <picture>
+                            <img
+                                src="../../../../img/homepage-hero-people-right.svg"
+                                alt="right" />
```
Comment (afnazu) [inline src/js/components/homepageUpdate/hero/Hero.jsx:65]: Should the alt text for these images be left blank and have role="presentation" instead? Regarding the accessibility AC on the ticket

**Review by afnazu — CHANGES_REQUESTED** [review]
Looks great!!! I just had a question regarding the accessibility AC

File: src/js/components/homepageUpdate/hero/Hero.jsx (hunk 1 above)
Comment (anjenkin) [reply src/js/components/homepageUpdate/hero/Hero.jsx:65]: Good questions, I think I should move that AC to the content ticket.

Comment (anjenkin) [reply src/js/components/homepageUpdate/hero/Hero.jsx:65]: Yeah, you're correct.  Fixing now.

**Review by afnazu — APPROVED** [review]

**Pull request merged**